### ISSUE TITLE
Background isn't blurred anymore from version 2.3

### ISSUE THREAD

**Issue by danielpetroianu** [AssistoLab/KVNProgress#85]
The background isn't blurred anymore. I've done some debugging and this bug was introduced at a741fea488a5c5a3500996b9500bc4f0645930b8 - Showing loader above keyboard.

Steps:
1. The progress window is created and makeKeyAndVisible (see `addProgressWindow`)
2. The `blurredScreenShot` method is called on the key windows , which now is the progress window, and the snapshot done by `- (BOOL)drawViewHierarchyInRect:afterScreenUpdates:` is empty.

**Comment by kevin-hirsch** [comment]
Hello @danielpetroianu!
Thanks for noticing this nasty bug 👍

It's now fixed. Until version 2.4 is released with this fix included, you can have it through CocoaPods by doing:
```ruby
pod 'KVNProgress', :git => 'https://github.com/AssistoLab/KVNProgress.git', :commit => '302514b'
```

**Comment by kevin-hirsch** [comment]
Fix available in version **2.3.1**.
You can use it via `pod 'KVNProgress'` ✨

**Comment by gravera** [comment]
I am still experiencing the same issue with 2.3.1.
